cloudservice/manager_test.go

```diff
@@ -236,7 +236,6 @@ var _ = Describe("New function", func() {
 func NewCloudServiceMock(isNewVersionMockValue bool, isNewVersionError error, convertToGenericIPRangesVersion string, convertToGenericIPRangesServiceList []string, convertToGenericIPRanges map[string][]string, convertToGenericIPRangesError error) *CloudProviderIPRangeManagerMock {
 	cloudProviderIPRangeMock := NewCloudProviderIPRangeManagerMock(GinkgoT())
 
-	cloudProviderIPRangeMock.EXPECT().IsNewVersion(mock.Anything, mock.Anything).Return(isNewVersionMockValue, isNewVersionError)
```

Comment:
Are you sure this call is not expected? Maybe they are an issue in the implementation or the coverage is not enough?

Reply:
I am yes, I looked at the code.
This is expected when there is already something stored.
Indeed, some tests might be missing